ambari-logsearch/ambari-logsearch-web/src/app/services/auth.service.ts

```diff
@@ -75,30 +76,48 @@ export class AuthService {
    * @param {string} password
    * @returns {Observable<Response>}
    */
-  login(username: string, password: string): Observable<Response> {
+  login(username: string, password: string): Observable<Boolean> {
     this.setLoginInProgressAppState(true);
     const response$ = this.httpClient.postFormData('login', {
       username: username,
       password: password
-    });
+    }).share();
```

Comment:
Would be better to add `share()` globally for all requests (`request()` method of `HttpClientService`). It used to be implemented in such a way some time ago. Any reason why this attempt doesn't fit?